documentation/BrowserChannel.md

```diff
@@ -24,39 +24,45 @@ Also, this component create notifications for every connected client at any time
 This small and documented client illustrates the workflow.
 
 ```javascript
-var io = require("socket.io-client");
+var io = require("ws");
 
 // The client starts the communication
 //
-var socket = io.connect("http://localhost:8081/browserChannel");
+var socket = new ws("ws://localhost:8081/browserChannel");
 
 // When the connection is done, the client tells to the flow manager its id
-//
-socket.on("connect", function () {
-    console.log("## Socket connected");
-    socket.send("org.chrome.cloud4chrome");
-});
 
-// Right after sending the id to the flow manager, the server will return back
-// the current settings in the system (if any)
-//
-socket.on("connectionSucceeded", function (settings) {
-    console.log("## Received the following settings: " + JSON.stringify(settings));
+socket.on("open", function () {
+    console.log("## Socket connected");
+    socket.send(JSON.stringify({
+        type: "connect",
```

Comment:
> But.... you presumably know all this since you have bound to my branch for your browser plugin work?

Yes, but I couldn't see any good reason why we should specify the 'type' field, I wasn't using it in my browser plugin work and the browserChannel was still accepting such connection requests. I guess that this is something we can improve in the upcoming browserChannel work, force clients to always use typed messages, and discarding them if it's not the case.